src/hostname/hostnamectl.c

```diff
@@ -128,22 +129,25 @@ static int print_status_info(StatusInfo *i) {
             !streq_ptr(i->hostname, i->static_hostname)) {
                 r = table_add_many(table,
                                    TABLE_FIELD, "Transient hostname",
-                                   TABLE_STRING, i->hostname);
+                                   TABLE_STRING, i->hostname,
+                                   TABLE_SET_COLOR, i->ansi_color);
```

Comment:
please add the "\1B[" prefix right before this here, please

(and the suffix too)